requirements.txt

```diff
@@ -9,3 +9,5 @@ Sphinx==1.4.8
 cryptography==1.4
 PyYAML==3.11
 codecov==2.0.5
+matplotlib==2.0.2
```

Comment:
It's a dependency for the new version of `scikit-image`, but it should be installed by `scikit-image` itself. I'll check and remove it if necessary.